src/wp-includes/general-template.php

```diff
@@ -696,6 +696,11 @@ function get_bloginfo( $show = '', $filter = 'raw' ) {
 			global $wp_version;
 			$output = $wp_version;
 			break;
+		case 'classicpress_version':
+		case 'classicpress_display_version': // May be different in the future.
```

Comment:
Let's not introduce new cases here where we don't need to. Version and display version should always be the same. Let's not write new deprecated code. Will save time going forward.